KDLoadingView/KDLoadingExtension.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  KDLoadingExtension.swift
+//  KDLoadingView
+//
+//  Created by Rodrigo Soldi Lopes on 02/03/17.
+//  Copyright © 2017 Kaique Damato. All rights reserved.
+//
+
+extension KDLoadingView {
+    
+    public class func animate(lineWidth: CGFloat = 2.0, size: CGFloat = 25, duration: CGFloat = 3.0, firstColor: UIColor? = #colorLiteral(red: 0, green: 0, blue: 0, alpha: 1), secondColor: UIColor? = nil, thirdColor: UIColor? = nil) {
+        guard let window = UIApplication.shared.keyWindow,
```

Comment:
Maybe it's better to split the guard let and put some log explaining the problem in the else statement.